stacker/context.py

```diff
@@ -10,13 +10,17 @@ class Context(object):
 
     """
 
-    _optional_keys = ('environment', 'stack_names', 'parameters', 'mappings',
-                      'config')
-
-    def __init__(self, namespace, **kwargs):
+    def __init__(self, namespace, environment=None, stack_names=None,
```

Comment:
Actually - I should probably get rid of kwargs now, since we don't do anything with it.  This will help with testing.